src/SingleArticleCard.jsx

```diff
@@ -32,19 +35,37 @@ export default function SingleArticleCard({ singleArticle }) {
         <h2>{title}</h2>
         <h4>{body}</h4>
         <h6>
-          author: {author} - topic: {topic} - created at:{" "}
-          {created_at.slice(0, 10)} - votes: {votes}
+          author: {author} <br />
```

Comment:
br tags should only be used when the line break is syntactically relevant, not for styling. And all of this shouldn't be wrapped in a header tag